### DISCUSSION TITLE
Translation Issues

### DISCUSSION THREAD

**Discussion by uli-on** [T8RIN/ImageToolbox#449] [General]
In the last set of strings there are misspellings etc in the English language. Cause I couldn't find a way to inform you via Weblate I post these here.

Key first
Misspelling/issue second

photo_picker_sub
adnroid

file_explorer_picker_sub
recieving 
pciked

add_sequence_number_sub
strandard

add_sequence_number_sub
(key uses „add“, string uses „replace“)

BTW, if you want a clean issue section you can probably shift this topic to Discussions. I can't open one in an inaccessable area, so I put it here.

**Comment by T8RIN** [comment]
Oh, thanks, i will fix it :)

**Comment by T8RIN** [comment]
You can fork the project and open a PR with updated strings, that would be great, because I don't have enough time now :(

**Comment by uli-on** [comment]
Not sure whether a closed topic sends mail. So perhaps leave open or transfer to discussions. 

Can you explain a little about the sense of the special orthography in sequenceNum and originalFilename? Is that a variable, so should be left untranslated?

Another typo, BTW, in string
load_image_from_net_sub
form - > from

**Comment by uli-on** [comment]
Also, does Weblate perhaps allow for activating and filling in a context field? See https://docs.weblate.org/ru/latest/user/translating.html#source-context and also the image right above, see field "explanation". Context would help a lot! Or screenshots of new UI elements.

**Comment by T8RIN** [comment]
> Not sure whether a closed topic sends mail. So perhaps leave open or transfer to discussions.
> 
> Can you explain a little about the sense of the special orthography in sequenceNum and originalFilename? Is that a variable, so should be left untranslated?
> 
> Another typo, BTW, in string load_image_from_net_sub form - > from

The last one is typo, first ones is needed to be translated, so everyone will get the meaning of each setting :)

**Comment by T8RIN** [comment]
Will fix everything

**Comment by T8RIN** [comment]
> Also, does Weblate perhaps allow for activating and filling in a context field? See https://docs.weblate.org/ru/latest/user/translating.html#source-context and also the image right above, see field "explanation". Context would help a lot! Or screenshots of new UI elements.

I will check what is it👀

**Comment by uli-on** [comment]
Thanks!

In the meantime, I think I found the two "variables" in the app's UI itself: These are used in the prefs drawer, for composing the example filename, right? So they should be as short as possible. These three informations (what, where, purpose (and - if suitable - how)) are often useful for us translators :)

If there's a direct communication channel to activate in Weblate itself, like Crowdin offers to its users, that'd be also great for translators for asking re specific strings.

**Comment by T8RIN** [comment]
No, unfortunately I have not heard of such a possibility (

**Comment by uli-on** [comment]
explicit_description:
"give" should be "given"

**Comment by T8RIN** [comment]
Thanks!
